core/deployment/src/main/java/io/quarkus/deployment/builditem/FeatureBuildItem.java

```diff
@@ -85,6 +85,7 @@
     public static final String SPRING_DATA_JPA = "spring-data-jpa";
     public static final String SPRING_SECURITY = "spring-security";
     public static final String SPRING_BOOT_PROPERTIES = "spring-boot-properties";
+    public static final String SPRING_CONFIG_SERVER_CLIENT = "spring-cloud-config-client";
```

Comment:
The variable name needs tpo be renamed?

Reply:
I'm never gonna catch a break with the rename :P

Reply:
Fixed